src/appleseed-max-impl/appleseedmetalmtl/appleseedmetalmtl.h

```diff
@@ -0,0 +1,209 @@
+
+//
+// This source file is part of appleseed.
+// Visit http://appleseedhq.net/ for additional information and resources.
+//
+// This software is released under the MIT license.
+//
+// Copyright (c) 2015-2017 Francois Beaune, The appleseedhq Organization
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+#pragma once
+
+// appleseed-max headers.
+#include "iappleseedmtl.h"
+
+// appleseed.foundation headers.
+#include "foundation/platform/windows.h"    // include before 3ds Max headers
+#include "foundation/utility/autoreleaseptr.h"
+
+// 3ds Max headers.
+#include <color.h>
+#include <IMaterialBrowserEntryInfo.h>
+#include <IMtlRender_Compatibility.h>
+#include <imtl.h>
+#include <interval.h>
+#include <iparamb2.h>
+#include <maxtypes.h>
+#include <ref.h>
+#include <strbasic.h>
+#undef base_type
+
+// Forward declarations.
+namespace renderer  { class Material; }
+class BaseInterface;
+class Bitmap;
+class Color;
+class FPInterface;
+class ILoad;
+class Interval;
```

Comment:
Either `#include <interval.h>` or the forward declaration of `Interval` is redundant here.